apps/team/src/pages/api/metadata/list.ts

```diff
@@ -1,6 +1,6 @@
-import { metadata } from "lib/models";
+import { metadata } from "@artsy-studio/models";
```

Comment:
Also wondering about this -- I guess more out of curiosity around keeping the api slender and constraining this sort of behavior at the app level vs something that assumed to be shared (unless of course yr certain it will be).

Reply:
This is a bit WIP, we'll revisit